src/event.rs

```diff
@@ -143,6 +144,133 @@ pub enum Event {
 		/// This will be `None` for events serialized by LDK Node v0.2.1 and prior.
 		reason: Option<ClosureReason>,
 	},
+	/// This event is emitted when we initiate a Payjoin transaction and before negotiating with
```

Comment:
In `start_request` we basically create the `original_psbt` we are going to share with the receiver, we also register it with `Filter` and other stuff (https://github.com/lightningdevkit/ldk-node/pull/295/files#diff-87bb691298fda83f0bcc2d9145208f7725ed0aca25764ef1fd1839002e2bc72aR79), after get response from `start_request` we start the background process(`PayjoinHandler::send_request`) waiting for response from the receiver. This event is basically indicating we are pending, waiting for a response. Maybe it should be called just before invoking `send_request` rather than in `start_request`